tools/project_manager/editproject.php

```diff
@@ -951,6 +970,14 @@ function show_visible_controls()
         }
         $this->row( _("Language"),                    'language_list',       $this->language         );
         $this->row( _("Genre"),                       'genre_list',          $this->genre            );
+
+        $project_charsuites = [];
+        if (isset($this->projectid))
+        {
+            $project = new Project($this->projectid);
+            $project_charsuites = $project->get_charsuites();
+        }
```

Comment:
We should probably add an `else` here that adds the basic-latin character suite to `$project_charsuites` for new projects 
 (if basic-latin is enabled on the site) for two reasons:
1. Upon initial roll-out we will only have basic-latin enabled and having it selected makes sense for 100% of the time.
2. After roll-out, it is very likely that for pgdp.net 99% of the projects will always use basic-latin.

One downside is that for the 1% of pgdp.net projects that don't use basic-latin they will have to unselect it, but that seems reasonable given the common case.

Reply:
Yeah I thought that might come up. I think it makes sense so defaulting to basic-latin now as it was before.